raycicmd/step_filter.go

```diff
@@ -43,6 +60,20 @@ func (f *stepFilter) hit(step *stepNode) bool {
 	return !f.reject(step) && f.accept(step)
 }
 
+func newSelectAndTagsStepFilter(
+	skipTags []string, selects []string, filterCmd []string,
+) (*stepFilter, error) {
+	select_filter := newSelectStepFilter(skipTags, selects)
+	tag_filter, err := newTagsStepFilter(skipTags, filterCmd)
+
+	return &stepFilter{
+		skipTags: skipTags,
+		selects:  select_filter.selects,
+		runAll:   tag_filter.runAll,
+		tags:     tag_filter.tags,
+	}, err
+}
```

Comment:
so, maybe just do this:

- have a `newStepFilter` that takes selects and tags
- rename `newTagsStepFilter` into `stepFilterFromCmd`, and when it gets the tags, call `newStepFilter()`?